lib/dialyzer/src/dialyzer_analysis_callgraph.erl

```diff
@@ -160,6 +151,21 @@ analysis_start(Parent, Analysis, LegalWarnings) ->
   Plt4 = dialyzer_plt:delete_list(Plt3, NonExportsList),
   send_analysis_done(Parent, Plt4, DocPlt).
 
+create_analysis_state({Plt, Analysis, Parent}, LegalWarnings, CServer) ->
+  #analysis_state{ codeserver = CServer
+                 , analysis_type = Analysis#analysis.type
```

Comment:
In my opinion, this comma style looks ugly. Perhaps more to the point, the rest of Dialyzer does not use this comma style, so it should not be used here.